src/data/mv_polynomial/monad.lean

```diff
@@ -194,7 +194,7 @@ by { ext1, apply bind₁_bind₁ }
 
 lemma bind₂_comp_bind₂ (f : R →+* mv_polynomial σ S) (g : S →+* mv_polynomial σ T) :
   (bind₂ g).comp (bind₂ f) = bind₂ ((bind₂ g).comp f) :=
-by { ext1; simp }
+by { ext; simp }
```

Comment:
You might want `ext : 2` or similar here to prevent it going further than needed